tests/ssg_test_suite/common.py

```diff
@@ -171,7 +173,9 @@ def _get_platform_cpes(platform):
             raise ValueError
         platform_cpes = set()
         for p in products:
-            p_cpes = ProductCPEs(p)
+            product_yaml_path = os.path.join(ssg_root, p, "product.yml")
```

Comment:
Here's where I was thinking a `get_product_yaml(product)` form would be most useful. Then it would be `load_product_yaml(path)` elsewhere. My 2c.